tests/unittest/_torch/multi_gpu/test_allreduce.py

```
@@ -261,16 +262,19 @@ def ref_residual_rms_norm_out_quant_nvfp4(x, res):
+    native_output = calc_func(xs[tensor_parallel_rank], residual)
     ref_output = ref_func(xs[tensor_parallel_rank], residual)
 
-    for calc_output_tensor, ref_output_tensor in zip(calc_output, ref_output):
-        check_accuracy(calc_output_tensor,
-                       ref_output_tensor,
-                       atol=0.05,
-                       rtol=0.15,
-                       percent=0.99)
+    for calc_output in (tuning_output, native_output):
+        for calc_output_tensor, ref_output_tensor in zip(
+                calc_output, ref_output):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)